src/webapp/src/client/app/common/expandable-menu/expandable-menu.component.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('simoonaApp.Common')
+        .component('aceExpandableMenu', {
+            replace: true,
+            transclude: true,
+            bindings: {
+                menuName: '<',
+                menuIconClass: '<',
+                menuTextTranslation: '<',
+                isExpanded: '<'
+            },
+            templateUrl: 'app/common/expandable-menu/expandable-menu.html',
+            controller: expandableMenuController,
+            controllerAs: 'vm'
+        });
+
+    function expandableMenuController() {
+        let vm = this;
+
+        vm.onExpandClick = onExpandClick;
+
+        const wasExpanded = localStorage.getItem(vm.menuName);
+        if (wasExpanded === 'true') {
+            vm.isExpanded = true;
```

Comment:
So the default (when nothing stored in localstorage yet) menu will be collapsed (i.e. `wasExpanded` will be undefined/false)?

As mentioned previously, unless you don't want to introduce some configuration for that (I would say it doesn't worth the effort) - we would like to keep default state for menus as expanded.
I agree that perhaps "External links" (usually the biggest menu group) could be collapsed by default, but it would be good to keep "Activities" and "Companies" as for instance expanded by default.

Reply:
Reversed the logic to expand everything by default.